scripts/filter-abund-single.py

```diff
@@ -154,6 +158,8 @@ def process_fn(record):
     # the filtering loop
     print('filtering', args.datafile, file=sys.stderr)
     outfile = os.path.basename(args.datafile) + '.abundfilt'
+    if args.outfile is not None:
```

Comment:
could you turn this into an if/else? I think it's nicer to be explicit.

also, on the next line, maybe change 'outfile = open(...)' to 'outfp = open(...)' - otherwise we're changing the type of object of outfile. (I probably put that in there, but now I don't like it ;)